contrib/inventory/ec2.py

```diff
@@ -592,65 +584,6 @@ def get_rds_instances_by_region(self, region):
                 error = "Looks like AWS RDS is down:\n%s" % e.message
             self.fail_with_error(error, 'getting RDS instances')
 
-    def include_rds_clusters_by_region(self, region):
```

Comment:
Looks like this merge will remove a different feature in ec2.py - can you fix this?